internal/proxy/highlighter.go

```diff
@@ -114,18 +115,26 @@ func (h *LexicalHighlighter) addTaskWithQuery(fieldID int64, query *highlightQue
 	})
 }
 
-func (h *LexicalHighlighter) AsSearchPipelineOperator(t *searchTask) (operator, error) {
+func (h *LexicalHighlighter) initHighlightQueries(t *searchTask) error {
```

Comment:
this initHighlightQueries is never called?

Reply:
Should called after create task. Some file not pushed.